plugins/jobs/server/models/job.py

```diff
@@ -63,24 +68,15 @@ def list(self, user=None, limit=0, offset=0, sort=None, currentUser=None):
         :param sort: The sort field.
         :param currentUser: User for access filtering.
         """
-        userId = user['_id'] if user else None
-        cursor = self.find({'userId': userId}, sort=sort)
-
-        for r in self.filterResultsByPermission(cursor=cursor, user=currentUser,
-                                                level=AccessType.READ,
-                                                limit=limit, offset=offset):
-            yield r
-
-    def listAll(self, limit=0, offset=0, sort=None, currentUser=None):
```

Comment:
Please read the below comment first for the background story. 
So, since I added this listall() recently myself, should I add listall() back with a deprecated mark?